src/test/java/pl/javastart/bootcamp/BootcampApplicationTests.java

```diff
@@ -13,6 +16,20 @@ public class BootcampApplicationTests {
 
     @Test
     public void contextLoads() {
+
     }
 
+    @Test
+    public void youtubeLinkShouldBeConverted() {
+        // given
+        String link1 = "https://www.youtube.com/watch?v=wyrGeEIMrcc";
+        String link2 = "https://youtu.be/wyrGeEIMrcc";
+
+        // when
+        String result = "wyrGeEIMrcc";
+
+        //then
+        assertEquals(YouTubeLinkConverter.convertLinksToOneFormat(link1), result);
+        assertEquals(YouTubeLinkConverter.convertLinksToOneFormat(link2), result);
+    }
```

Comment:
- Podziel te testy na 2 (każda asercja w osobnym teście)
- testy powinny znajdować się w klasie nowej klasie `YouTubeLinkConverterTest`
- są to testy jednostkowe, więc nie powinno być w tej klasie adnotacji `@SpringBootTest`
- w sekcji `when` powinno być wywołanie metody `convertLinksToOneFormat` i przypisanie wyniku do zmiennej, a nie deklaracja zakładanego wyniku (to robilmy w `then`)